sites/ademe/src/frontend/scss/settings/_colors.scss

```diff
@@ -423,6 +412,8 @@ $r-theme: (
     title-color: r-color('dark'),
     organization-color: r-color('tealish'),
     organization-media-empty-background: r-color('white-three'),
+    organization-shadow: 0 0 6px r-color('silver'),
+    code-color: r-color('battleship-grey'),
```

Comment:
Should we add this instruction to UPGRADE.md?

Reply:
There is no mention about these variables within the UPGRADE.md but if these variables are missing, sass fails to compile so we cannot forgot them.